kore/src/main/scala/org/kframework/Strategy.scala

```diff
@@ -63,7 +63,7 @@ class Strategy(heatCool: Boolean) {
 
           import rich._
           
-          if (!defn.mainModule.importedModuleNames.contains("STRATEGY$SYNTAX") || r.att.contains("anywhere") || r.att.contains("macro")) {
+          if (!defn.mainModule.importedModuleNames.contains("STRATEGY$SYNTAX") || r.att.contains("anywhere") || r.att.contains("macro") || r.att.contains("alias") || r.att.contains("macro-rec") || r.att.contains("alias-rec")) {
```

Comment:
I suppose we can't use `ExpandMacros.isMacro` here because that lives in `kernel`.

Reply:
Correct. Really this code should be rewritten in java and live in the kernel, but that's a bigger change than I want to make right now.